tests/validators/rules/test_progress_value_source_validation.py

```diff
@@ -0,0 +1,116 @@
+import json
+
+from structlog import configure, getLogger
+from structlog.stdlib import LoggerFactory
+
+from app.validators.questionnaire_validator import QuestionnaireValidator
+
+logger = getLogger()
+
+configure(logger_factory=LoggerFactory())
+
+
+def _open_and_load_schema_file(file_path):
+    with open(file_path, encoding="utf8") as json_file:
+        return json.load(json_file)
```

Comment:
Minor, move the existing ones into a utils file and reuse the method instead of duplicating.